docs/_docs/framework/capabilities/understand/reporting.md

```diff
@@ -152,8 +152,8 @@ This capability is a part of the FinOps Framework by the FinOps Foundation, a no
 
 You can also find related videos on the FinOps Foundation YouTube channel:
 
-<!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->
-{% include video.html title="Reporting and analytics videos" id="CVTJLdcozj1eEpxT" list="PLUSCToibAswlDSQdehKhi7ysP2hmetigl" %}
+<!--[!VIDEO https://www.youtube.com/embed/pD9NeBOvspU?list=PLUSCToibAswlDSQdehKhi7ysP2hmetigl&pp=iAQB]-->
```

Comment:
Maybe this one relates a bit more: https://www.youtube.com/watch?v=2qEcRzx26VA

Reply:
All of these links are to playlists for the respective capabilities. Ultimately, I'd like to link to playlists we own, but we can do that later.